src/components/index.ts

```diff
@@ -56,6 +56,7 @@ export * from './utils/withEventBrokerDomHandlers';
 export * from './utils/useBodyScrollLock';
 export * from './utils/useEventBroker';
 export * from './utils/useLayer';
+export * from './utils/LayerManager';
```

Comment:
This file has exported private types. We don't want to expose them all

Reply:
fixed